src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Security/UseDefaultDllImportSearchPathsAttribute.cs

```diff
@@ -79,21 +90,25 @@ public override void Initialize(AnalysisContext context)
                     defaultValue: UnsafeBits);
                 var defaultDllImportSearchPathsAttributeOnAssembly = compilation.Assembly.GetAttributes().FirstOrDefault(o => o.AttributeClass.Equals(defaultDllImportSearchPathsAttributeTypeSymbol));
 
+                // Does not analyze local functions. To analyze local functions, we'll need to use RegisterSyntaxAction.
```

Comment:
I'm okay with this decision. We should update the doc to mention that local functions will not be analyzed if others are okay with this.

Reply:
Just to be clear, you mean this doc, right? https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca5392

Reply:
Probably both that and https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca5393